test/functional/wallet_createwallet.py

```diff
@@ -26,6 +26,12 @@ def run_test(self):
         node = self.nodes[0]
         self.generate(node, 1) # Leave IBD for sethdseed
 
+        self.log.info("Run createwallet with invalid parameters.")
+        # Run createwallet with invalid parameters. This must not prevent a new wallet with the same name from being created with the correct parameters.
+        # See PR #24859 for details.
```

Comment:
I don't think we need to add obscure `#nnn` to the source code. If anything this should be a full url, but I don't think it adds any value, so best to remove it.

Reply:
+1

Reply:
Done.